protocols/kad/src/query/peers/closest.rs

```diff
@@ -487,7 +486,7 @@ mod tests {
             let known_closest_peers = random_peers(g.gen_range(1, 60), g)
                 .into_iter()
                 .map(Key::from);
-            let target = Key::from(Into::<Multihash>::into(PeerId::random()));
```

Comment:
I know it is just a small technicality, but using a `Key<Multihash>` in these tests reflects common usage better. This should just be a `Key` based on a random `Multihash`, not sure why the current code goes through a `PeerId`.

Reply:
Again the problem mentioned in the other comment that I don't find a way to get `Key<Multihash>` working.